### DISCUSSION TITLE
G6.Util.processParallelEdges处理平行边不生效

### DISCUSSION THREAD

**Discussion by AmieJing** [antvis/G6#3377] [Q&A] [answered]
设置了G6.Util.processParallelEdges处理平行边，但在2s后新增一条平行边，该设置不生效;
复现demo：https://codesandbox.io/s/fast-shadow-h4gdw?file=/src/components/Topo.vue
![image](https://user-images.githubusercontent.com/37952540/146509906-fc850b67-3113-4588-a3b7-8c14f867b22b.png)

**Comment by Yanyan-Wang** [comment]
因为你是在上面处理的 this.data.edges。addItem 新增边之后，要重新处理一下图上的数据。

**Comment by AmieJing** [comment, marked as answer]
是的，新增边之后，需要重新处理平行边，可以了，谢谢
![image](https://user-images.githubusercontent.com/37952540/147028166-8f68e300-c0d5-4cde-ad32-dd43badbd2ac.png)
